cyborg/accelerator/drivers/peripheral/pi_gpio.py

```diff
@@ -48,6 +48,9 @@ def discover_peripherals(self):
                     "devices": [
                         {"type": "c", "path": GPIO_DEVICE},
                     ],
-                }
+                    "capabilities": {
+                        "effective": ["CAP_SYS_RAWIO"],
```

Comment:
We might want to update this to also set it for "inherited", "permitted", and "bounding". Docker appears to do this when you specify `--cap-add`. In particular, "effective" capabilities can only be set from the "permitted" list.